AddTrelloComment/README.md

```diff
@@ -24,3 +24,15 @@ To obtain them you need to follow the [Trello instructions](https://developer.at
             TRELLO-TOKEN: ${{ secrets.TRELLO_TOKEN }}
                     
 ```
+### Pull Request
+If the action is used in a GitHub pull request workflow, by passing the html_url and request body, you  the action will create a link in the Trello card which is a link back to the GitHub pull request
```

Comment:
Typo: `you the action will create a link`

Reply:
resolved